converters/toZigbee.js

```diff
@@ -746,6 +746,33 @@ const converters = {
             await entity.command('genIdentify', 'identifyTime', {identifytime: value}, getOptions(meta));
         },
     },
+    ZNCLDJ11LM_ZNCLDJ12LM_options: {
+        key: ['options'],
+        convertSet: async (entity, key, value, meta) => {
+            const opts = {
+                'direction': 'original',
+                'hand_open': true,
+                'reset_move': false,
+                ...value,
+            };
+
+            const payload = [
+                0x07,
+                0x00,
+                opts.reset_move ? 0x01: 0x02,
+                0x00,
+                opts.direction == 'reverse' ? 0x01: 0x00,
+                0x04,
+                !opts.hand_open ? 0x01: 0x00,
+                0x12,
+            ];
+            meta.logger.info(`ZNCLDJ11LM convertSet ${JSON.stringify(payload)}`);
```

Comment:
This should log something human friendly, otherwise change level to `meta.logger.debug`.